subgraph.yaml

```diff
@@ -93,11 +93,11 @@ dataSources:
           handler: handleVoteCast
       file: ./src/pooltogether_mapping.ts
   - kind: ethereum/contract
-    name: UniswapGovernorAlpha
```

Comment:
we can't remove alpha, there were votes there as well. We have to support both alpha and bravo. Look at Arthur's changes for example